packages/core/content-manager/admin/src/history/components/HistoryAction.tsx

```diff
@@ -1,28 +1,39 @@
-import { useQueryParams } from '@strapi/admin/strapi-admin';
+import { useQueryParams, useTracking } from '@strapi/admin/strapi-admin';
 import { ClockCounterClockwise } from '@strapi/icons';
 import { stringify } from 'qs';
 import { useIntl } from 'react-intl';
-import { useNavigate } from 'react-router-dom';
+import { useNavigate, useLocation } from 'react-router-dom';
 
 import type { DocumentActionComponent } from '../../content-manager';
 
 const HistoryAction: DocumentActionComponent = ({ model, document }) => {
   const { formatMessage } = useIntl();
   const [{ query }] = useQueryParams<{ plugins?: Record<string, unknown> }>();
   const navigate = useNavigate();
+  const { trackUsage } = useTracking();
+  const { pathname } = useLocation();
   const pluginsQueryParams = stringify({ plugins: query.plugins }, { encode: false });
 
   if (!window.strapi.features.isEnabled('cms-content-history')) {
     return null;
   }
 
+  const handleOnClick = () => {
+    const destination = { pathname: 'history', search: pluginsQueryParams };
+    trackUsage('willNavigate', {
+      from: pathname,
+      to: `${pathname}/${destination.pathname}?${destination.search}`,
```

Comment:
just wondering (cause I might do the same in another pr) do we normally track the search params?

Reply:
Hello @Marc-Roig thanks a lot for your question...I discussed the topic with @yanniskadiri and we tried to check if there are other "willNavigate" where we pass the search params, there is an example when you click on the API tokens in the Setting navbar
![Screenshot 2024-11-12 alle 16 11 30](https://github.com/user-attachments/assets/968ca33e-e4e5-4b16-9096-72a4959e9de6)
That said we think there is no real value passing the search param in the case of history, because we want to measure how many people click on the button so we decided to remove the search params part...I hope it can be useful in your case too

Reply:
noice!! thank you for sharing, I will keep the search params out of the preview event too! ty